R/map.R

```diff
@@ -137,54 +139,53 @@
 #'   map(summary) |>
 #'   map_dbl("r.squared")
 #'
-#' @examplesIf interactive() && requireNamespace("mirai", quietly = TRUE) && requireNamespace("carrier", quietly = TRUE)
+#' @examplesIf interactive() && requireNamespace("mirai", quietly = TRUE)
 #' # Run in interactive sessions only as spawns additional processes
 #'
 #' # To use parallelized map, set daemons (number of parallel processes) first:
 #' mirai::daemons(2)
 #'
-#' mtcars |> map_dbl(sum, .parallel = TRUE)
+#' mtcars |> map_dbl(parallelize(\(x) sum(x)))
 #'
 #' 1:10 |>
-#'   map(function(x) stats::rnorm(10, mean = x), .parallel = TRUE) |>
-#'   map_dbl(mean, .parallel = TRUE)
+#'   map(parallelize(\(x) stats::rnorm(10, mean = x))) |>
+#'   map_dbl(parallelize(\(x) mean(x)))
 #'
```

Comment:
Can you please include an example where you include some other state from the current environment?

Reply:
Have added our infamous `slow_lm()` example in [5f023e6](https://github.com/tidyverse/purrr/pull/1185/commits/5f023e6dcc7f046080e6c1c4fdbb162acca510ea). 😄